Formula/pandoc.rb

```diff
@@ -22,11 +22,6 @@ class Pandoc < Formula
   depends_on "gmp"
```

Comment:
Thanks for providing the reference. I haven't tried it on GNU/Linux but on Mac OS X I don't notice such an issue without installing `gmp`. 

```
$ pandoc -o test.html test.md      # works on all my machines
```

Also, `pandoc` doesn't link with `gmp` even with `depends_on "gmp"` in the formula.

```
$ otool -L `which pandoc`
/usr/local/bin/pandoc:
	/usr/lib/libz.1.dylib (compatibility version 1.0.0, current version 1.2.5)
	/usr/lib/libiconv.2.dylib (compatibility version 7.0.0, current version 7.0.0)
	/usr/lib/libSystem.B.dylib (compatibility version 1.0.0, current version 1226.10.1)
```

I have just now built the current version `1.17.1` without `depends_on "gmp"`. The generated binary works fine.

Here are the logs, https://gist.github.com/d0235638893c6074fa80e906589a0232